cmd/cluster/core/create.go

```diff
@@ -167,7 +168,7 @@ type KubevirtPlatformCreateOptions struct {
 }
 
 type AWSPlatformOptions struct {
-	AWSCredentialsFile      string
+	AWSCredentialsOpts      awsutil.AWSCredentialsOptions
```

Comment:
You do not need to change this now, but FWIW if you had embedded the `awsutil.AWSCredentialsOptions` it would have required a much smaller diff (and a lot less typing!)